content/local-setup.md

```diff
@@ -0,0 +1,359 @@
+# Local Setup
+
+# Lesson Objectives
+
+- Set up a local environment for Solana program development
+- Use basic Solana CLI commands
+- Run a local test validator
+- Use Rust and the Solana CLI to deploy a Solana program from your local development environment
+- Use the Solana CLI to view program logs
+
+# TL;DR
+
+- To get started with Solana locally, you’ll first need to install **Rust** and the **Solana CLI**
+- Using the Solana CLI you can run a **local test validator** using the `solana-test-validator` command
+- Once you have Rust and Solana CLI installed, you’ll be able to build and deploy your programs locally using the `cargo build-bpf` and `solana program deploy` commands
+- You can view program logs using the `solana logs` command
+
+# Overview
+
+So far in this course, we've used Solana Playground to develop and deploy Solana programs. And while it's a great tool, for certain complex projects you may prefer to have a local development environment set up. This may be in order to use crates not supported by Solana Playground, to take advantage of custom scripts or tooling you've created, or simply out of personal preference.
+
+With that said, this lesson will be slightly different from the others. Instead of covering a lot of ground on how to write a program or interact with the Solana network, this lesson will primarily focus on the admittedly tedious task of setting up your local development environment.
+
+In order to build, test, and deploy Solana programs from your machine, you'll need to install the Rust compiler and the Solana Command Line Interface (CLI). We'll start by guiding you through these installation processes, then cover how to use what you'll have just installed.
+
+The installation instructions below contain the steps for installing Rust and the Solana CLI at the time of writing. They may have changed by the time you're reading this, so if you run into issues please consult the official installation pages for each:
+- [Install Rust](https://www.rust-lang.org/tools/install)
+- [Install the Solana Tool Suite](https://docs.solana.com/cli/install-solana-cli-tools)
+
+## Setup on Windows (with Linux)
+
+### Download Windows Subsystem for Linux (WSL)
+
+If you are on a windows computer, it is recommended to use Windows Subsystem for Linux (WSL) to build your Solana Programs.
```

Comment:
@jamesrp13 "windows" ---> "Windows" (there are three instances of "windows" (here on line 34, and again on lines 36 and 48

Reply:
Will fix and push a new commit